splinepy/helpme/create.py

```diff
@@ -298,6 +298,337 @@ def revolved(
     return type(spline)(**spline_dict)
 
 
+def swept(
+    cross_section,
+    trajectory,
+    cross_section_normal=None,
+    set_on_trajectory=False,
+    rotation_adaption=None,
+):
+    """
+    Sweeps a cross-section along a trajectory. The cross-section
+    receives rotation into the direction of the trajectory tangent
+    vector and is then placed either at the evaluation points of the
+    trajectory's knots or at the trajectory's control points. This
+    depends on the value of the set_on_trajectory parameter. It can
+    create both a surface or a solid, depending on the dimension of
+    the cross-section.
+
+    The sweeping process has some limitations, since the cross-section
+    cannot be preserved exactly along the whole trajectory.
+
+    The maths behind can be found in the NURBS Book, Piegl & Tiller,
```

Comment:
Please mention which variant you implemented here and how you determine ambiguous values